src/modules/universal-discovery/module-usage.md

```diff
@@ -0,0 +1,106 @@
+    - **maxHeight** _{Number}_ - a maximum height of panel container,
+    - **id** _{String}_ - panel identifier,
+    - **startingLocationId** _{Number}_ - location ID,
+    - **findLocationsByParentLocationId** _{Function}_ - finds locations related to the parent location,
+    - **findContentBySearchQuery** _{Function}_ - finds content matching a given text query,
+    - **contentTypesMap** _{Object}_ - content types map with content type ids as keys,
+    - **multiple** _{Boolean}_ - can select multiple content items flag,
+    - **labels** _{Object}_ - a hash containing text messages to be placed across many places in a component,
+- **attrs** {Object} - any optional list of props that should applied to the panel component. It can override the panel props listed above.
+
+## Props list
+
+The `<UniversalDiscoveryModule />` module can handle additional properties. There are 2 types of properties: **required** and **optional**. All of them are listed below.
+
+### Required props
+
+Without all the following properties the Universal Discovery module will not work.
+
+**onConfirm** _{Function}_ - a callback to be invoked when a user clicks on the confirm button in a Universal Discovery popup. The function takes one param: `content` which is an array of content items structs.
+
+**onCancel** _{Function}_ - a callback to be invoked when a user clicks on the cancel button in a Universal Discovery popup. It takes no extrz params.
+
+### Optional props
+
+Optionally, Universal Discovery module can take a following list of props:
+
+**loadContentInfo** _{Function}_ - loads content info. It takes 2 params: `contentId` and `callback`
+**loadContentTypes** _{Function}_ - loads content types data. It takes one param: `callback`,
+**canSelectContent** _{Function}_ - checks whether a content item can be selected. It takes one param: `content` - the content struct,
+**findContentBySearchQuery** _{Function}_ - finds a content using a search query. It takes 2 params: `query` and `callback`,
+**findLocationsByParentLocationId** _{Function}_ - finds sub items of a given location. It takes 2 params: `parentLocationId` and `callback`,
+**title** _{String}_ - the title of Universal Discovery popup. Default value: `Find content`,
+**multiple** _{Boolean}_ - can select multiple content items flag. Default value: `true`,
+**activeTab** _{String}_ - active tab identifier. Default value: `browse`,
+**startingLocationId** _{Number}_ - location ID. Default value: `1`,
+**maxHeight** _{Number}_ - maximum height of panel container. Default value: `500`,
+**searchResultsPerPage** _{Number}_ - max amount of items visible per page in the search results. Default value: `10`,
+**extraTabs** _{Array}_ - optional, extra tabs. Each tab definition is an object containing the following properties (all of them are required):
+    - **id** _{String}_ - unique tab identifier (it cannot be neither: `browse` nor `search`),
```

Comment:
`    - **id** _{String}_ - unique tab identifier (it cannot be `browse` or `search`),`